pkg/runtime/runtime.go

```diff
@@ -1075,12 +1075,24 @@ func (a *DaprRuntime) initNameResolution() error {
 
 func (a *DaprRuntime) publishMessageHTTP(msg *pubsub.NewMessage) error {
 	var cloudEvent map[string]interface{}
+
 	err := a.json.Unmarshal(msg.Data, &cloudEvent)
 	if err != nil {
 		log.Debug(errors.Errorf("failed to deserialize cloudevent: %s", err))
 		return err
 	}
 
+	contentType := cloudEvent[pubsub.DataContentTypeField].(string)
```

Comment:
We need to check if the `DataContentTypeField` exists in the map before the string conversion, otherwise cloud events that do not have this property will cause the sidecar to crash